### PR TITLE
feat: add lit-element/open-wc hello world

### LINKED ISSUE #61: Lit-element/Open-wc hello world
By making a PR you are agreeing to that contributing a hello world project allows the repo/project owner to use the hello world app in examples for their own use.

- [x] only the most simple basic hello world app
- [x] remove all tests and commands for tests and dependencies for tests
- [x] must say "Hello World" on the main component
- [x] only include the minimal components (1 ideally)
- [x] no routing, not extras whatsoever
- [x] no styling
- [x] nothing else !
- [x] must work by running `npm install` and `npm start` (or equivalent)
- [x] must be in the folder named `<tech-name>`
- [x] add yourself using the all-contributors bot `@allcontributors please add @<your-github-account> for code`
- [x] add build steps and which folder to serve in the README.md, Serve the `public` folder

### PR DESCRIPTION
**Pull request by nweldev**
Closes #61

By making a PR you are agreeing to that contributing a hello world project allows the repo/project owner to use the hello world app in examples for their own use.

- [x] only the most simple basic hello world app
- [x] remove all tests and commands for tests and dependencies for tests
- [x] must say "Hello World" on the main component
- [x] only include the minimal components (1 ideally)
- [x] no routing, not extras whatsoever
- [x] no styling
- [x] nothing else !
- [x] must work by running `npm install` and `npm start` (or equivalent)
- [x] must be in the folder named `<tech-name>`
- [x] add yourself using the all-contributors bot `@allcontributors please add @<your-github-account> for code`
- [x] add build steps and which folder to serve in the README.md, Serve the `public` folder

### CODE REVIEW

**Comment by nweldev** [comment]
@allcontributors please add @noelmace for code

File: lit-element/index.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  <meta name="Description" content="Webcomponent hello-world following open-wc recommendations.">
+  <base href="/">
+
+  <style>
+    html,
+    body {
+      margin: 0;
+      padding: 0;
+      font-family: sans-serif;
+      background-color: #ededed;
```
Comment (devpato) [inline lit-element/index.html]: I believe the rules for PR say they shouldn't be styles

File: lit-element/src/HelloWorld.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { LitElement, html, css } from 'lit-element';
```
Comment (devpato) [inline lit-element/src/HelloWorld.js]: if one day esLint rules get added to the project, this file can have linter issues for declared variables and not use them. E.g **css** is not being used anywhere.

File: lit-element/src/hello-world.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import { HelloWorld } from './HelloWorld.js';
```
Comment (devpato) [inline lit-element/src/hello-world.js]: This comment is just a thought, but based on the convention used in the demos from the lit-element team, this file should be named` index.js` and the `HelloWorld.js` as `hello-world.js`

**Comment by devpato** [comment]
I'm glad someone submitted LitElement!! Thank you sir!

File: lit-element/index.html (hunk 1 above)
Comment (nweldev) [reply lit-element/index.html]: Thx, I forgot about this one.

File: lit-element/src/HelloWorld.js (hunk 2 above)
Comment (nweldev) [reply lit-element/src/HelloWorld.js]: Yep, css again :smile: Done.

File: lit-element/src/hello-world.js (hunk 3 above)
Comment (nweldev) [reply lit-element/src/hello-world.js]: This was generated by Open-WC. So I don't know what would be better: keeping the by default naming, or `index.js`. Let me know what you decide.

**Review by johnpapa — APPROVED** [review]

**Pull request merged**